### PR TITLE
AB#78096 - ability to hide vertical menu by default

### PR DESCRIPTION
**Pull request by LenaelleL**
# Description

Adds a selection in the app settings to show or not the vertical menu by default. Not available if vertical menu is not selected.
Did it for 2.1.x but should have done it for beta I guess, sorry

## Useful links

- [ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/78096)
- Link to back-end : https://github.com/ReliefApplications/oort-backend/pull/772

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Tested with both new and old apps

## Screenshots

image
image

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [ ] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/back-office/src/app/application/pages/settings/settings.component.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h1 class="!m-0">{{ 'common.general' | translate }}</h1>
         [minRows]="2"
         [maxRows]="5"
         name="textarea"
-      >
+      >=)
```
Comment (RenzoPrats) [inline apps/back-office/src/app/application/pages/settings/settings.component.html:56]: I think it's a misspelling
```suggestion
      >
```

**Review by RenzoPrats — CHANGES_REQUESTED** [review]
Hi @LenaelleRelief, I found this issue I think it's a bug 
Peek 12-10-2023 16-33
When we edit some setting and save it, we are not able to modify and save again only if we reload the page.

**Comment by LenaelleL** [comment]
Hi ! @RenzoPrats thanks a lot ! I indeed noticed this bug but since it was also impacting the "side menu" selector, I thought it was outside the scope of this PR, that's good if you did it anyway aha

Can you confirm that with this solution, it is possible to update the "side menu" selection again after already saving it ?

**Comment by AntoineRelief** [comment]
> Hi ! @RenzoPrats thanks a lot ! I indeed noticed this bug but since it was also impacting the "side menu" selector, I thought it was outside the scope of this PR, that's good if you did it anyway aha
> 
> Can you confirm that with this solution, it is possible to update the "side menu" selection again after already saving it ?

it's out of scope of this PR
as it was not based on correct branch, the issue you described is solved in beta branch

**Review by RenzoPrats — APPROVED** [review]
Ah okay, I didn't know, so despite of that is working fine!

**Comment by AntoineRelief** [comment]
@LenaelleRelief 
closing this one
I'll push similar changes directly on beta
thanks!

**Pull request closed without merging**